libbeat/scripts/generate_fields_docs.py

```diff
@@ -1,10 +1,12 @@
 import yaml
+import sys
 
-def document_fields(output, section, SECTIONS, level=3):
+
+def document_fields(output, section, SECTIONS):
 
     if "anchor" in section:
         output.write("[[exported-fields-{}]]\n".format(section["anchor"]))
-    output.write("{} {} Fields\n\n".format(level*'=', section["name"]))
+    output.write("=== {} Fields\n\n".format(section["name"]))
```

Comment:
Be aware, that the beats had different implementations here:

with === and levels
* winlogbeat: https://github.com/elastic/beats/pull/890/files#diff-7ad2172f5c2f9e52b7c9c5bc6802e56aL22
* filebeat: https://github.com/elastic/beats/pull/890/files#diff-fb727cf14fe8bf8169f4ab4d7cce8cc6R7

with {} and without levels:
* packetbeat: https://github.com/elastic/beats/pull/890/files#diff-78627e2210ed02aff6673f49d77ea60dL28

with === but without levels:
* topbeat: https://github.com/elastic/beats/pull/890/files#diff-4089deb253efa80c804e59ed8af3a1a4L23

I tried to unify these behaviours with the levels.